site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -234,7 +234,11 @@ If you have a FusionAuth deployment and want to delete it, do so by https://acco
 
 [WARNING.warning]
 ====
-Make sure you have a backup. When a deployment is destroyed, there is no way to retrieve its data. All FusionAuth Cloud backups are destroyed as well.
+Before destroying a deployment, ensure you back up your user data and configuration. 
```

Comment:
Sorry, late on this one. But on cloud , since there is not an export, do we need to say this must be done via APIs (calling get or writing down config)?

And I think you have a typo, the last sentence says business does not include backup, I think you mean basic.